board/bcm/bcm7252_full_defconfig

```diff
@@ -205,6 +205,7 @@ CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
 CONFIG_USB_STORAGE=y
+CONFIG_USB_USBNET=y
```

Comment:
Is this needed for the Nexus power management lib?

Reply:
It is to support Ethernet over USB